Portfolio.html

```diff
@@ -4,11 +4,85 @@
+            </div>
+        </div>
+    </div>
+</div>
+    <div class="blog-wrapper">
+                <div class="main-content-wrapper">
+                <div class="blog-container" id="C2">
+                    <div class="sub-heading-rule-before"></div>
+                    <h2>THREE PEDALS ONLY</h2>
+                    <div class="sub-heading-rule-after"></div>
+                    <p class="main-content-text">Lots of interesting stuff here about manual gearboxes and how manual gearboxes are dying out and why this is such a bad thing for car enthusiasts everywhere and it's better for a driver's car to have a manual than an auto.</p>
+                    <div class="blog-text">
+                        <img class="gears" src="if_f085_213059.png" alt="gears">
+                        <p class="blog-text-paragraph">Stuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxes</p>
+                        <img class="gearbox" src="https://user-images.githubusercontent.com/271652/38042847-6d825426-3283-11e8-8b37-bb58514c3a2f.png" alt="stick shift">
+                        <p class="blog-text-paragraph">Stuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxes</p>
+                    </div>
+                </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
```

Comment:
This needs to be at the top of your code before any other elements.

Reply:
done.